components/brave_rewards/common/constants.h

```diff
@@ -0,0 +1,29 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_COMMON_CONSTANTS_H_
+#define BRAVE_COMPONENTS_BRAVE_REWARDS_COMMON_CONSTANTS_H_
+
+namespace brave_rewards {
+
+extern const char kBatPaymentMethod[];
+
+namespace errors {
+
+extern const char kBatTransactionFailed[];
```

Comment:
you can put all of them into header with `constexpr`

Reply:
resolved